announcementsAPI.json

```diff
@@ -288,6 +296,45 @@
         }
       }
     },
+    "/announcements/{announcementId}/like": {
+      "put": {
+        "tags": [
+          "AnnouncementsEndpoints"
+        ],
+        "summary": "Adds or deletes announcement from liked announcements depending on boolean value in parameters. Requires adopter role. Gets adopter id from auth claims.",
+        "parameters": [
+         {
+            "name": "announcementId",
+            "in": "path",
+            "required": true,
+            "style": "simple",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "isLiked",
+            "in": "query",
+            "required": true,
+            "style": "form",
+            "schema": {
+              "type": "boolean"
+            }
+          }
```

Comment:
Maybe we could have that in body? Either as an object with `isLiked` field, or just as a `true/false` in body. Not too important though, it's fine either way IMO.

Reply:
Nvm, I didn't see that there already was some discussion about this